src/wallet/api/wallet_manager.cpp

```diff
@@ -81,6 +81,12 @@ bool WalletManagerImpl::closeWallet(Wallet *wallet)
 
 bool WalletManagerImpl::walletExists(const std::string &path)
 {
+    bool keys_file_exists;
+    bool wallet_file_exists;
+    tools::wallet2::wallet_exists(path, keys_file_exists, wallet_file_exists);
+    if(keys_file_exists){
+        return true;
+    }
```

Comment:
If you're going to change this, squash. Adding a commit to change trivial stuff in the same PR just adds spam. We aim for easy to review. You don't *have* to change it though, I OK'd it. Though the gratuitous whitespace below would be nice to fix :D